src/couch_eval/src/couch_eval.erl

```diff
@@ -86,6 +94,26 @@ release_map_context({ApiMod, Ctx}) ->
 map_docs({ApiMod, Ctx}, Docs) ->
     ApiMod:map_docs(Ctx, Docs).
 
+-spec acquire_context(language()) -> {ok, context()} | {error, any()}.
```

Comment:
The way function is written it cannot return `{error, any()}`. I think the correct spec would be:

```
-type error(_Error) :: no_return().

....

spec acquire_context(language()) -> 
   {ok, context()} 
   | error({invalid_eval_api_mod, Language :: binary()})
   | error({unknown_eval_api_language, Language :: binary()}).
```

Reply:
Don't forget to update `-callback acquire_context()....` above as well.